src/config/dynamic-theme-fixes.config

```diff
@@ -22488,15 +22488,19 @@ INVERT
 
 ================================
 
+lafundacionscp.wikidot.com
```

Comment:
We currently sort fixes by the first URL in the sub-list. Thus the order is broken now.

Reply:
does it right?